_posts/2019-11-08-method-references-the-unpolished-gem-of-java-8.markdown

```diff
@@ -174,7 +174,7 @@ dogs.stream()
 
 Except…
 
-> Take all dogs for which not vegetarian says they are healthy
+> Take all dogs for which: not veterinarian says they are healthy
```

Comment:
I believe you meant veterinarian. The colon is just to be clear where you start speaking about the code above

Reply:
Oh right hahaha!

As for the colon, the sentence is meant to be difficult to read (;